tests/network/services.go

```diff
@@ -46,12 +46,15 @@ import (
 var _ = SIGDescribe("[Serial]Services", func() {
 	var virtClient kubecli.KubevirtClient
 
-	runTCPClientExpectingHelloWorldFromServer := func(host, port, namespace string, isIPv6 bool) *batchv1.Job {
+	runTCPClientExpectingHelloWorldFromServer := func(host, port, namespace string, isIPv6 bool, retries int32) *batchv1.Job {
 		var pingCmd string
 		if isIPv6 {
 			pingCmd = fmt.Sprintf("ping -c1 %s;", host)
 		}
-		job, err := virtClient.BatchV1().Jobs(namespace).Create(context.Background(), tests.NewHelloWorldJob(host, port, pingCmd), k8smetav1.CreateOptions{})
+		job := tests.NewHelloWorldJob(host, port, pingCmd)
+		job.Spec.BackoffLimit = &retries
```

Comment:
My only question here is if this retries parameter should be made available in the original question, but I understand why you only change this private helper. I'm OK with it (I would probably have done the same actually ...).

Reply:
I can prepare a follow up with passing retries around in functions, but in locking 1.20 did have more priority.

Reply:
Just a thought, I'm ok either way.